cli/pkg/utils/utils.go

```diff
@@ -45,12 +47,16 @@ func SetupNode(dir string) error {
 
 	// Check for nvm
 	if os.Getenv("NVM_DIR") != "" {
-		cmd = exec.Command("bash", "-l", "-c", "$NVM_DIR/nvm.sh", "use")
+		cmd = exec.Command("bash", "-l", "-c", ". $NVM_DIR/nvm.sh && nvm use")
```

Comment:
I realized that the first command `$NVM/nvm.sh` just loads nvm in the terminal. The following `use` call is also needed.